measures/ResStockArgumentsPostHPXML/measure.rb

```diff
@@ -98,13 +105,17 @@ def run(model, runner, user_arguments)
     # Process each building
     doc_buildings.each_with_index do |building, index|
       hpxml_bldg = @hpxml.buildings[index]
-      if hpxml_bldg.hvac_controls.to_a.length == 0
-        runner.registerInfo("Skipping hvac flexibility for building #{index + 1} since it has no HVAC controls.")
-        next
+      if hpxml_bldg.hvac_controls.to_a.length != 0 && !skip_hvac_flexibility?(args)
+        hvac_schedule = create_hvac_schedule(index)
+        modified_schedule = modify_hvac_schedule(index, hvac_schedule)
+        write_schedule(modified_schedule, building, index, output_csv_path)
+      end
+      if args[:ev_flex_enabled] && !skip_ev_flexibility?(args)
```

Comment:
Does this not check the same thing twice?

Reply:
Good catch!